src/container.tsx

```diff
@@ -8,7 +8,10 @@ import { RootState } from './store';
 
 export interface PublicProperties {
   provider: any;
-  route: string;
+  route: {
```

Comment:
I suggest to create a separate type for `route`
```
interface Route {
    znsRoute: string;
    app: string;
}
```
we can use the type in the code below